src/CentreonRemote/Application/Webservice/CentreonConfigurationRemote.php

```diff
@@ -540,42 +542,54 @@ public function authorize($action, $user, $isInternal = false)
      * @param bool $noProxy to do not use configured proxy
      */
     private function addServerToListOfRemotes(
```

Comment:
I think the name could be something like "addOrUpdateRemoteServer" and maybe you could split this method in 2 smaller methods : addRemoteServer and updateRemoteServer